scripts/billing/sql/forecast_net_balance.sql

```diff
@@ -0,0 +1,114 @@
+/*
+Query returns the forecasted future balance for the next month given
+historical usage trends
+*/
+
+SELECT
+  month,
+  balance,
+  future_balance
+FROM (
+  SELECT
+    FORMAT_TIMESTAMP('%Y%m', usage_end_time) AS month,
+    SUM(cost) + SUM((SELECT SUM(amount) FROM UNNEST(credits))) AS balance,
+    NULL AS future_balance
+  FROM `data-analytics-pocs.public.billing_dashboard_export`
+  WHERE
+    FORMAT_TIMESTAMP('%Y%m', usage_end_time) 
+      >= FORMAT_DATE('%Y%m', DATE_ADD(CURRENT_DATE(), INTERVAL -3 MONTH))
+    AND FORMAT_TIMESTAMP('%Y%m', usage_end_time) 
+      != FORMAT_TIMESTAMP('%Y%m', CURRENT_TIMESTAMP())
+  GROUP BY month
+  ORDER BY month
+)
+UNION ALL (
+  SELECT
+    FORMAT_TIMESTAMP('%Y%m', CURRENT_TIMESTAMP()) AS month,
+    NULL AS balance,
+    SLOPE * (EXTRACT(YEAR FROM CURRENT_TIMESTAMP()) * 12
+      + EXTRACT(MONTH FROM CURRENT_TIMESTAMP())) + INTERCEPT AS future_balance
+  FROM (
+    SELECT 
+      SLOPE,
```

Comment:
In this `SELECT` clause and ones below, the columns are aliased with UPPER_CASE but other files and within this file, columns are aliased with lower_case. Please change the UPPER_CASE aliases to lower_case to be consistent.